packages/test/test-end-to-end-tests/src/test/gc/gcSweepAttachmentBlobs.spec.ts

```diff
@@ -807,6 +807,7 @@ describeNoCompat("GC attachment blob sweep tests", (getTestObjectProvider) => {
 
 				// Connect the container after the blob is uploaded. Send an op to transition the container to write mode.
 				mainContainer.connect();
+				await provider.ensureSynchronized();
```

Comment:
why do you need to modify these other tests? doesn't seem like it should be necessary

Reply:
Assert "blob handles should be different" was failing. Not sure about how de-duplication works but I believe previously pending list was able to get rid of blobHandle1 at this point just by connecting (or maybe after waiting for write mode). We won't do that now until the blob is acked and attached.

Reply:
Actually while fixing blobManager.spec.ts I see something odd when we upload while disconnected. I'm debugging

Reply:
Errors in blobManager.spec.ts were unrelated to this

Reply:
what is the test error if you remove these? it shouldn't be necessary to modify these tests. if it is necessary we should know exactly why, otherwise i worry we are changing something we don't understand, which could lead to a bug.

Reply:
the assert below fails: "blob handles should be different".

Reply:
Indeed it was a bug.